add-chain/testdata/monorepo/op-node/rollup.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Not sure if we discussed this already, but did we decide not to include `genesis.json` input files and associated checks in the tests? If so, perhaps best to record the reason why in a readme. The readme can also explain choices such as the separate module.

Reply:
I'm not sure what variations of `genesis.json` and `rollup.json` are expected. Are you able to point me to any reference material that shows all the variations of those inputs that `add-chain` should be able to successfully process?

Reply:
I don't know about reference material, but here is a doc which covers some genesis changes impacting the `registry-data` tool which we could in theory track through the tests in this module. It's another thing we could circle back on.

Reply:
I think your comment is missing a link? Agreed that we could circle back on this. This PR mainly aimed to convert the existing code from shell script to go to allow us to more easily handle some of these types of changes. Rather than making all of the upgrades in the same PR, I think it would be best to address those upgrades as follow-up work

Reply:
Agreed. Here is the link I forgot https://www.notion.so/oplabs/Superchain-Genesis-Hash-checks-60e173e9fb0c43e88a81d0666cb3d582

Reply:
Issue created [here](https://github.com/ethereum-optimism/superchain-registry/issues/220) to capture that work.